### DISCUSSION TITLE
Creating subscriptions

### DISCUSSION THREAD

**Discussion by helloalbin** [flutter-stripe/flutter_stripe#1177] [Q&A]
How do I create a subscription with flutter stripe? This is what I have done so far.

1. Created a customer from backend
2. Created a payment intent from backend
3. Inititalized a payment sheet
4. presented payment sheet
5. Confirm payment sheet payment

A payment is successful, but there is no way I can attach this payment to the customer to create the subscription.  

Kindly help. I am stuck here.

**Comment by fldvlpr** [comment]
@helloalbin Any updates on this one. I'm also following the same way. How can we make the subscription payment attach to a subscription or product?

Also how to fetch the list of all purchased subscriptions of a customer can be retrieved?

Also how to cancel a subscription of the customer?

**Comment by helloalbin** [comment]
I have created a custom form to collect the credit card information and used the https://api.stripe.com/v1/payment_methods API to create the payment method. It works for cards that has no additional security measures, but won't work for cards that has additional security like 3D secure. 

At the moment, I am exploring some alternate options for Stripe.

**Reply by devDonald** [reply]
Any sample code on this implementation?
